features/events/traps/src/test/java/org/opennms/netmgt/trapd/TrapdListenerBlueprintIT.java

```diff
@@ -66,7 +75,19 @@
 	private final TrapNotificationLatch m_handler = new TrapNotificationLatch(3);
 
 	private int m_port;
-
+	
+        private TrapdConfigBean trapConfiguration;
```

Comment:
Ok,This has been addressed.I thought i can reuse this.Now i have reverted back the changes and written as suggested.